Calculator/Calculator_base.cpp

```diff
@@ -200,32 +201,63 @@ int CCalculator::parseQuestion()
 		case '-':
 		case '*':
 		case '/':
+
 		{
 			// 연산자 전까지 확인한 값을 집어넣음
-			double newnumber = atof(numberInput.c_str());
+			if (numberInput == (string)"")
```

Comment:
* wstring 중에 비어있는지 여부를 확인하는 함수를 사용해 주세요.

```suggestion
			if (numberInput.empty())
```
```suggestion
			if (numberInput.length() == 0)
```